packages/dds/merge-tree/src/properties.ts

```diff
@@ -110,17 +124,23 @@ export function addProperties(
 	oldProps: PropertySet | undefined,
 	newProps: PropertySet,
 ): PropertySet {
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
 	const _oldProps = oldProps ?? createMap<any>();
 	extend(_oldProps, newProps);
 	return { ..._oldProps };
 }
 
 /**
+ * Replace values of undefined in one PropertySet with values for the same key from another PropertySet.
```

Comment:
(Not for this PR) these functions are both `@deprecated` and `@internal`. Is there anything blocking us from removing them? Are there work items tracking this?

Reply:
Created [AB#8760](https://dev.azure.com/fluidframework/internal/_workitems/edit/8760) to track the removals.